### PR TITLE
docs(am): risk-based alert documentation

### PR DESCRIPTION
**Pull request by remisultan**
**Issue**

https://github.com/gravitee-io/issues/issues/7638

**Description**

Documentation for risk-based alerts in AM

🚀 CI was able to deploy the build of this PR, so you can now try it directly [here](https://graviteedocs.blob.core.windows.net/docs-am-risk-assessment-alerts/index.html)

### CODE REVIEW

**Review by ashraf706 — APPROVED** [review]

File: pages/am/3.x/user-guide/alerts/user-guide-alerts.adoc [hunk 1]
```diff
@@ -25,6 +25,7 @@ The following table lists the available alert types.
 |===
 |Type|Key|Description
 | Too many login failures |too_many_login_failures| Alert when the number of login failures is abnormally high.
+| Risk-based alerts |risk_assessment| Alert when the user behaviour seem suspicious.
```
Comment (leleueri) [inline pages/am/3.x/user-guide/alerts/user-guide-alerts.adoc:28]: maybe we can mark 'too many reset password', 'too many locked out... as not implemented ?

**Review by leleueri — CHANGES_REQUESTED** [review]

File: pages/am/3.x/user-guide/alerts/user-guide-alerts-notifications-partial.adoc [hunk 2]
```diff
@@ -50,7 +49,19 @@ These properties vary depending on the rules configured and the type of event be
 |===
 |Key |Description
 
+|`environnement`| Current organisation with property `id`.
```
Comment (Johngren) [inline pages/am/3.x/user-guide/alerts/user-guide-alerts-notifications-partial.adoc:52]: environment*

File: pages/am/3.x/user-guide/alerts/user-guide-alerts.adoc (hunk 1 above)
Comment (remisultan) [reply pages/am/3.x/user-guide/alerts/user-guide-alerts.adoc]: FP

File: pages/am/3.x/user-guide/alerts/user-guide-alerts-notifications-partial.adoc (hunk 2 above)
Comment (remisultan) [reply pages/am/3.x/user-guide/alerts/user-guide-alerts-notifications-partial.adoc]: FP

**Review by tcompiegne — APPROVED** [review]

**Pull request merged**